### ISSUE TITLE
Chat Styling

### ISSUE THREAD

**Issue by rohrlach** [upstage-org/limelight#127]
This issue proposes that we more distinctly represent various types of dialog in the chat window so that we can use a higher number of variations.  To begin with:
![screenshot from 2017-05-09 14-16-10](https://cloud.githubusercontent.com/assets/2830313/25865823/4ceb5e70-34c2-11e7-90db-9a2f327bf696.png)

**Comment by rohrlach** [comment]
Deploy in 40 mins

**Comment by helenvarley** [comment]
i agree that it's good to be able to distinguish between different types of chat input, however i think we need to be quite careful with this & definitely make it optional. the chat window is important but it needs to work with what's going on visually on the stage - which is often quite a lot - & not clash with it or dominate (unless of course that is wanted by a particular performance).

we also need to consider how much chat is visible in the window; for example, when we are projecting UpStage for a gathered audience, we usually increase the font size so that it's more readable from a distance. given that there are currently only 8 lines of chat showing on my screen, i can see there would be even less than that if the font size was increased. 

even when the font size hasn't been increased, having less lines of chat visible will be problematic for performances where there is sometimes a lot of chat going on. people won't be able to keep up with it, & even if you can scroll up & down, it's not great if you have to do it a lot, it makes it difficult to keep up & respond.

can there be options within the stage set-up, where you can choose whether to have plain text or styled in the chat?

**Comment by vickis** [comment]
oops should have commented about the colours here 
i quite like it 

the synaeathesists should be consulted!
